### PR TITLE
Add supported ES versions to the table

### PR DESCRIPTION
**Pull request by misiekhardcore**
## What is the purpose of the change

Ive added missing supported elastic search versions to the table

### CODE REVIEW

**Comment by akeller** [comment]
Hi @misiekhardcore! Thanks for your contribution. Given that this looks like it's for current and future docs, would you mind including the changes in `/docs/` as well? 

I've also added @RomanJRW for an engineering review. I did not add Christina for a technical writing review since changes like this can merge with an appropriate engineering review.

**Comment by akeller** [comment]
Should this also include updates for 3.9.0 based on #1258?

**Comment by akeller** [comment]
@pepopowitz FYI, another Optimize PR, but this file is in the Reference section. This doesn't need to move to the Optimize instance, correct?

**Comment by pepopowitz** [comment]
@akeller correct, this one is not affected by the Optimize changes.

File: versioned_docs/version-8.0/reference/supported-environments.md [hunk 1]
```diff
@@ -62,11 +62,11 @@ _Note: You can use newer Modeler versions with older Zeebe versions too._
...
 
-| Improve        | Automate                                   | Java version              |
-| -------------- | ------------------------------------------ | ------------------------- |
-| Optimize 3.3.x | Camunda Platform 7.12.11+, 7.13.5+, 7.14.x | OpenJDK 8+ or OpenJDK 11+ |
-| Optimize 3.4.x | Camunda Platform 7.13.5+, 7.14.x, 7.15.x   | OpenJDK 8+ or OpenJDK 11+ |
-| Optimize 3.5.x | Camunda Platform 7.13.5+, 7.14.x, 7.15.x   | OpenJDK 11+               |
-| Optimize 3.6.x | Camunda Platform 7.14.x, 7.15.x, 7.16.x    | OpenJDK 11+               |
-| Optimize 3.7.x | Camunda Platform 7.14.x, 7.15.x, 7.16.x    | OpenJDK 11+               |
-| Optimize 3.8.x | Camunda Platform 7.15.x, 7.16.x, 7.17.x    | OpenJDK 11+               |
+| Improve        | Automate                                   | Java version              | Elasticsearch version                                                         |
```
Comment (RomanJRW) [inline versioned_docs/version-8.0/reference/supported-environments.md:65]: why do we have the table headers as `Improve` and `Automate` out of interest? I can't think of a good reason for this 🤔

**Review by RomanJRW — APPROVED** [review]
LGTM, just one comment on something that isn't new 🙂

**Comment by akeller** [comment]
@misiekhardcore are you able to revisit this PR?

**Comment by misiekhardcore** [comment]
@akeller sorry for the delay. I've added those elasticsearch versions to `/docs/` as well

**Review by akeller — APPROVED** [review]

**Pull request merged**